### ISSUE TITLE
[Update] : [Adsense] : Thin Content Issue 

### ISSUE THREAD

**Issue by h0lybyte** [KBVE/kbve.com#676]
**Describe the update**

One of the current plans was to update the website so that we could get approval for Google Adsense. 
One of the major reasons that we get denied for their approval was because the website had thin content, which we should address.

* * *

**References for update**
Include any links / data for the update that must be done.

[Google Adsense Support](https://support.google.com/adsense/thread/24001628/why-am-i-getting-thin-content-with-little-or-no-added-value-response?hl=en) that breaks down some of the issues that it could be. 

* * *

**Security/Performance risks**
Are there any major security and/or performance risks?!
There are no major security or performance risks.

* * *

**Unrelated**

I am also making this issue to track the n8n automation, but that is un-related.

* * *

**Comment by h0lybyte** [comment]
Automating Tasker
* * *
[KBVE Support](https://kbve.com/support/)
* * *
[KBVE Discord](https://kbve.com/discord/)
* * *
[KBVE Staff](https://kbve.com/team/)
* * *
[KBVE VIP Priority](https://kbve.com/vip/) || [KBVE Community](https://kbve.com/c/)
* * *
[HubSpot Ticket Created](https://app.hubspot.com/contacts/20854121/ticket/1745038189)
* * *

**Comment by h0lybyte** [comment]
Not much of an update on this topic, I will move it over to Postpone because it involves a bunch of other issues to be addressed before we can touch back on this.

**Comment by h0lybyte** [comment]
```
Dear Ad Manager Partner,
--
 
On 30 August 2023, Google will update Shocking content in the Google Publisher Restrictions to include an exception for gameplay imagery. In the context of gameplay imagery, content is only considered to “contain gruesome, graphic, or disgusting accounts or imagery” or “depict acts of violence” if it depicts acts of torture, sexual violence, violence against minors, violence against prominent real-name persons, or violence against an individual or group on the basis of a characteristic that is associated with systemic discrimination or marginalization.
 
This update will take effect on 30 August 2023.

```
 
Please review the update to determine if it affects your site or app. If your site or app received restricted ad serving previously due to Shocking content but you believe ad serving should no longer be restricted following the update, you may request a review or appeal of your site or app on or after 30 August 2023. Learn more about requesting a review or appeal.
 
Until next time,
The Google Ad Manager Team


</body>
</html>Dear Ad Manager Partner,

On 30 August 2023, Google will update [Shocking content](https://notifications.google.com/g/p/ADa0GC8dDBQeUYfgX-5AfcTFw_8TUmoTp_394McI4BTaZGIlhn7WJ4CmIVUkByLlaD5TJ0vsK4bOsQpR8Wce1z1mYpySw_DAr8SvYIQaEV_A3POLN6pQqoDhRlm4dhI8uO95dnZg6HR7JS-0_Fr9F37SYsyoG9Kbo17RmwbzcfC7INg8dLwb_2pR1jnxaBq7tvPSWCjmYCU8yyEc2_OiGOBZVxICEAwMZ1qNqyiQ9WthQsFr1mayNin-R7KFuo1YC4e1SxDCmtCU3zolZt4) in the Google Publisher Restrictions to include an exception for gameplay imagery. In the context of gameplay imagery, content is only considered to “contain gruesome, graphic, or disgusting accounts or imagery” or “depict acts of violence” if it depicts acts of torture, sexual violence, violence against minors, violence against prominent real-name persons, or violence against an individual or group on the basis of a characteristic that is associated with systemic discrimination or marginalization.

This update will take effect on 30 August 2023.

Please review the update to determine if it affects your site or app. If your site or app received restricted ad serving previously due to Shocking content but you believe ad serving should no longer be restricted following the update, you may request a review or appeal of your site or app on or after 30 August 2023. Learn more about [requesting a review or appeal](https://notifications.google.com/g/p/ADa0GC-m8Yd6sULKy1ar-sSL9H_DCk5y9_6lJcEHKNkj62Ry9dgKP-_LpugQYMiG3-C7ZEAglfTZZZ15ly7sWFe5z3NlkL7M9Ed_x8wjLRXVjIFSHEPjUNg4a8jlCDLSIxDKiekrkKBBrAGApHDw2LCsQYfme3d-SaMNDlV_KS3tEvWFVIbxdRcq7PlsS6tpsQVxnuV4YRGZP3IkmXnznnwP9QTCUGx7d8kQwtIVT7Sym06vTa6cWYWFijEXTr1gPkYfqO5nGztYunjf26YgRL--O-MshGJ7FzM-K_mT5uCPtBOdL0lobhf9LETH-KYg5A).

Until next time,
The Google Ad Manager Team

**Comment by h0lybyte** [comment]
Thin content issues will be resolved during the official v1 launch of KBVE. https://github.com/KBVE/kbve/issues/105
